### ISSUE TITLE
The progress animation not show sometimes

### ISSUE THREAD

**Issue by 4Oranges** [dteoh/SlowQuitApps#6]
![screen shot 2016-09-22 at 11 59 50 am](https://cloud.githubusercontent.com/assets/7879714/18736709/4a70c2da-80bc-11e6-9890-83bd335b87b9.png)

Here's a screenshot for this issue. Sometimes the animation inside the border does not display correctly, however the app still works properly.

I don't know what cause this problem. May be it happened only after I have launched some apps. If so, It should be more than one apps would cause this problem. 

After this issue had occurs, the SlowQuitApps stuck on the status. Neither relaunch the SlowQuitApps nor close all other apps would not help. Only reboot can bring the animation back.

**Comment by dteoh** [comment]
Yeah, it is a known issue. It was something that I noticed happening only from 10.9+. On 10.8 it was perfect. I guess I should review all the APIs I'm using to see if something is being misused. Triggering this bug is not something that I can reproduce on demand, which makes trying to fix this a nightmare.

I'm surprised you need to reboot, though. This makes the bug even more worrying. I've had the app eventually "recover" on its own, so it is a bit bizarre to me.

**Comment by 4Oranges** [comment]
I'have upgrade to macOS 10.12 about a week and it sometimes "fixed itself".  And this "recover" never happened to me when I was on OS X 10.11.

**Comment by dteoh** [comment]
Sorry for the slow progress on this issue. I have just published a new release which might fix the problem. Try it out and let me know if it still happens.

**Comment by 4Oranges** [comment]
Here you can see a slightly shadow at the beginning of the sector, on OS X 10.11.6. I'll test it on OS X 10.12 latter
<img width="308" alt="screen shot 2016-12-26 at 14 45 14" src="https://cloud.githubusercontent.com/assets/7879714/21477176/7906f566-cb7a-11e6-8d1e-97767215985b.png">
<img width="19" alt="screen shot 2016-12-26 at 14 45 14 2" src="https://cloud.githubusercontent.com/assets/7879714/21477181/805c11a2-cb7a-11e6-9f33-fd4dbc446402.png">

**Comment by 4Oranges** [comment]
And the size of the shadowed sector seems to be a random value

**Comment by dteoh** [comment]
Yeah, that sub-shadow is a known issue.

**Comment by eSoares** [comment]
Just happened to me, even after killing the process and opening again the problem continues.

**Comment by dteoh** [comment]
Do you all happen to use some sort of global keyboard modification system like Karabiner?

**Comment by eSoares** [comment]
I use BetterTouchTool.

**Comment by eSoares** [comment]
Just happened to have a probably related bug: don't drawing anything, but still working (same time to Command+Q to exit open app).
Going to resource monitor, killing it and open again works.

**Comment by eSoares** [comment]
Just found that adding the app to login items and reboot, the app starts but doesn't draw (similar problem to the above).

Added on boot via login items, more info: http://osxdaily.com/2006/11/29/how-to-launch-application-on-system-start-in-mac-os-x/

**Comment by dteoh** [comment]
At this stage, I can only guess that there is some sort of software incompatibility going on.

The only global shortcut program I am using concurrently with SQA is Spectacle. I don't use any automatic window management programs. I haven't personally experienced any drawing issues since the last published update, so this is reaching into the very hard to debug territory. 

The best path forward is to try disabling extra helpers or customizations that could collide with the handling of the Command Q key combo, and seeing if that has any effect or not.

**Comment by dteoh** [comment]
I've released v0.2.8, which may fix things.

Closing in favour of issue #16.
